bin/rucio

```diff
@@ -1796,6 +1795,97 @@ def list_datasets_rse(args):
+    dids_list = []
+    containers = []
+    datasets = []
+    error_types = ['Total DIDs', 'DID not submitted because it is a file', 'DID not submitted because it is a container', 'DID not submitted because it is not part of the lifetime campaign', 'DID successfully submitted']
+    summary = {1: [], 2: [], 3: [], 4: []}
+    for did in dids:
+        scope, name = get_scope(did, client)
+        dids_list.append({'scope': scope, 'name': name})
+    summary[0] = dids_list
+    for meta in client.get_metadata_bulk(dids_list):
+        scope = meta['scope']
+        name = meta['name']
+        if meta['did_type'] == 'FILE':
+            logger.warning('%s:%s is a file. Will be ignored' % (scope, name))
+            dids_list.remove({'scope': scope, 'name': name})
+            summary[1].append({'scope': scope, 'name': name})
+        elif meta['did_type'] == 'CONTAINER':
+            logger.warning('%s:%s is a container. It needs to be resolved' % (scope, name))
+            containers.append({'scope': scope, 'name': name})
+            dids_list.remove({'scope': scope, 'name': name})
+            summary[2].append({'scope': scope, 'name': name})
+        else:
+            if not meta['eol_at']:
+                logger.warning('%s:%s is not affected by the lifetime model' % (scope, name))
+                dids_list.remove({'scope': scope, 'name': name})
+                summary[3].append({'scope': scope, 'name': name})
+            else:
+                logger.info('%s:%s will be declared' % (scope, name))
+                datasets.append({'scope': scope, 'name': name})
+                dids_list.remove({'scope': scope, 'name': name})
+                summary[4].append({'scope': scope, 'name': name})
+    for did in dids_list:
+        scope = did['scope']
+        name = did['name']
+        logger.warning('%s:%s does not exist' % (scope, name))
+
+    if containers:
+        logger.error('One or more DIDs are containers. They must be resolved into a list of datasets to request exception. Full list below')
+        for container in containers:
+            logger.error('%s:%s' % (container['scope'], container['name']))
+        logger.error('Please take action. Submission aborted')
+        return FAILURE
+    if datasets:
+        try:
+            client.add_exception(dids=datasets, account=client.account, pattern='', comments=reason, expires_at=expiration)
+        except UnsupportedOperation as err:
+            logger.error(err)
+            return FAILURE
+        except Exception:
+            logger.error('Failure to submit exception. Please retry')
+            logger.debug(traceback.format_exc())
+            return FAILURE
+        logger.info('Exception successfully submitted. Summary below')
+        for cnt, error in enumerate(error_types):
+            print('{0:100} {1:6d}'.format(error, len(summary[cnt])))
+        return SUCCESS
+
+    else:
+        logger.error('Nothing to submit')
+        return SUCCESS
```

Comment:
This creates an unnecessary indentation for `if datasets:`.
I would suggest an early exit at line 1868:
```pyhton
it not datasets:
  logger.error('Nothing to submit')
  return SUCCESS

# This is the case in which datasets contains items.
...
```